group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -1307,13 +1307,14 @@ private CoordinatorResult<ConsumerGroupHeartbeatResponseData, Record> consumerGr
             }
         }
 
+        // The subscription metadata is updated in two cases:
+        // 1) The member has updated its subscriptions;
+        // 2) The refresh deadline has been reached.
+        Map<String, Integer> subscribedTopicNamesMap = group.subscribedTopicNames();
```

Comment:
nit: Should we use `subscribedTopicNames` too?

Reply:
we can't cause the list is also named subscribedTopicNames right?